docs/parts/wokwi-led.md

```diff
@@ -16,18 +16,18 @@ Standard 5mm LED.
 
 ## Attributes
 
-| Name       | Description                     | Default value        |
-| ---------- | ------------------------------- | -------------------- |
-| color      | The color of the LED button     | "red"                |
-| lightColor | The color of the light          | depends on the color |
-| label      | Text that appears below the led |                      |
+| Name       | Description                     | Default value                      |
+| ---------- | ------------------------------- | ---------------------------------- |
+| color      | The color of the LED button     | "red"                              |
+| lightColor | The color of the light          | the value of the "color" attribute |
```

Comment:
Actually, it's not the value of "color" attribute, it's derived from it:

https://github.com/wokwi/wokwi-elements/blob/7c59476aedcd820d87dc17b6e2ffd8939c4432f2/src/led-element.ts#L4

For colors that do not appear on that list, the default value is "ff8080"